get-involved.php

```diff
@@ -112,38 +94,74 @@
 
 <section id="desktop-development" class="dark">
     <div class="grid">
-        <div class="half">
-            <h2>Desktop Development</h2>
+        <div class="two-thirds">
+            <h1>Desktop Development</h1>
             <p>Our desktop environment and all its apps are built using Vala, GTK+, Clutter, Cairo, Granite, and a number of other free libraries. All of our code is hosted on Launchpad.net, a free service for open source projects. We're always looking for contributors of all skill levels.</p>
-
+        </div>
+        <div class="whole">
+            <h2>Our Progress for The Loki Alpha1 Milestone:</h2>
```

Comment:
Alpha1 -> Alpha 1

Side note: considering beta is when we release a download, wouldn't it be more informative to show our status to beta 1?